js/views/modals/listingDetail/Listing.js

```diff
@@ -168,13 +282,30 @@ export default class extends BaseModal {
         shipsFreeToMe: this.shipsFreeToMe,
         ownListing: this.model.isOwnListing,
         displayCurrency: app.settings.get('localCurrency'),
+        shipsFromCountry: app.settings.get('country'),
       }));
 
       super.render();
 
       this._$deleteListing = null;
       this._$shipsFreeBanner = null;
       this._$popInMessages = null;
+      this._$photoSection = null;
+
+      // commented out until variants are available
+      // this.$('.js-variantSelect').select2();
+
+      this.$photoSelected = this.$('.js-photoSelected');
+      this.$photoSelected.backgroundDraggable();
+
+      const shippingDest = this.$('#shippingDestinations');
+
+      shippingDest.select2({
+        data: this.select2CountryData,
+        placeholder: app.polyglot.t('listingDetail.shipToPlaceholder'),
+      });
+
+      shippingDest.val(app.settings.get('country')).trigger('change');
```

Comment:
Do we need to trigger the event here? Can't we just directly call the function here? You'd probably need to change the function signature so it accepts the data it needs rather than a click handler and then seperate the click handler into a separate function that calls the other one. Seems like a much cleaner way to me than synthetically firing a change event.